Tests/Foundation/Tests/TestDecimal.swift

```diff
@@ -99,6 +99,10 @@ class TestDecimal: XCTestCase {
         XCTAssertEqual(.lossOfPrecision, NSDecimalAdd(&result, &one, &addend, .plain), "1 + 1e-39")
         XCTAssertEqual("1", result.description)
         XCTAssertEqual(.orderedSame, NSDecimalCompare(&one, &result), "1 + 1e-39")
+
+        let a = Decimal(string: "498.7509045")!
+        let b = Decimal(string: "8.453441368210501065891847765109162027")!
+        XCTAssertEqual(a + b, Decimal(string: "507.2043458682105010658918477651091")!)
```

Comment:
Rather than using force unwrap, mark the test function as `throws` and use:
`let a = try XCTUnwrap(Decimal(string: "498.7509045"))` etc.